app.js

```diff
@@ -0,0 +1,10 @@
+const express = require ('express');
+const app= express();
+const fs = require ('fs/promises');
```

Comment:
What is this "fs" being used for? This would be used to access the file system and is not needed for this project.